RELEASE/scripts/autoscend/iotms/mr2021.ash

```diff
@@ -774,7 +778,7 @@ int auto_fireExtinguisherCharges()
 // returns zone specific skill if in usable zone and hasn't been used yet there this ascension. Otherwise returns empty string
 string auto_FireExtinguisherCombatString(location place)
 {
-	if(auto_fireExtinguisherCharges() < 20)
+	if(auto_fireExtinguisherCharges() < 20 || !auto_is_valid($skill[Fire Extinguisher: Zone Specific]))

```

Comment:
whats the reasoning of adding this?

Reply:
If we can't use skills, like the professor, we shouldn't try to use the Extingo. I added a change to auto_is_valid to check whether you are a professor or not.